src/backend/joanie/core/enums.py

```diff
@@ -72,6 +76,22 @@
         ORDER_STATE_VALIDATED,
         pgettext_lazy("As in: the order is validated.", "Validated"),
     ),
+    (
+        ORDER_STATE_PENDING_PAYMENT,
+        pgettext_lazy("As in: the order payment is pending.", "Pending payment"),
+    ),
+    (
+        ORDER_STATE_FAILED_PAYMENT,
+        pgettext_lazy("As in: the last order payment failed.", "Failed payment"),
```

Comment:
```suggestion
        pgettext_lazy("As in: the last order payment has failed.", "Failed payment"),
```

To be consistent with the other states.